src/components/OpenTofuNotice.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import Admonition from "@theme/Admonition"
+
+export const OpenTofuNotice = () => {
+    return (
+        <div>
+            <Admonition type="tip" title="OpenTofu Support">
```

Comment:
I'm not sure it's a "tip" per se, but the green is nice enough and stands out so I'm okay to roll with it!